soccer_vision_msgs/msg/Goalpost.msg

```diff
@@ -0,0 +1,42 @@
+# Describes a detected goalpost from vision modules, in image coordinates.
+
+uint8 SIDE_UNKNOWN=0
+uint8 SIDE_LEFT=1
```

Comment:
Detecting whether or not this is the left or right goal posts seems doable in image space and I like the default of 0 so that if you don't care it is no extra code and you have a valid semantic.